nodes/nodes/eco-poison-pill-operator.adoc

```diff
@@ -28,3 +28,4 @@ include::modules/eco-poison-pill-operator-troubleshooting.adoc[leveloffset=+1]
 == Additional resources
 
 * The Poison Pill Operator is supported in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
```

Comment:
link doesn't work

Reply:
@prabinovRedhat the links seem to be working fine. Did you access it from the preview links that i povided in the first comment of the PR?
http://file.rdu.redhat.com/avbhatt/ent-bz-2109524/nodes/nodes/eco-poison-pill-operator.html#daemon-set-exists_poison-pill-operator-remediate-nodes

Reply:
works, thanks